tests/testlib/s2n_extension_test_utils.c

```diff
@@ -24,3 +25,20 @@ int s2n_connection_allow_all_response_extensions(struct s2n_connection *conn)
     POSIX_CHECKED_MEMSET(&conn->extension_requests_sent, 0xFF, S2N_SUPPORTED_EXTENSIONS_BITFIELD_LEN);
     return S2N_SUCCESS;
 }
+
+int s2n_connection_mark_response_extension_received(struct s2n_connection *conn, uint16_t iana_value)
+{
+    s2n_extension_type_id extension_id = s2n_unsupported_extension;
+    POSIX_GUARD(s2n_extension_supported_iana_value_to_id(iana_value, &extension_id));
+    S2N_CBIT_SET(conn->extension_responses_received, extension_id);
```

Comment:
I suppose you could also do just "s2n_connection_mark_extension_received" and set the bit for both requests and responses received. Might be more useful in the future.

Reply:
Okay yeah, that makes sense. I changed that in 6e35809